internal/operators/mce/config.go

```diff
@@ -13,6 +36,34 @@ const (
 	SNOMinimumCpu    int64 = 8
 )
 
-type Config struct {
-	MceMinOpenshiftVersion string `envconfig:"MCE_MIN_OPENSHIFT_VERSION" default:"4.10.0"`
+func ParseMCEConfig(config Config) (*ParsedConfig, error) {
```

Comment:
```suggestion
func parseMCEConfig(config Config) (*ParsedConfig, error) {
```
no need to expose it outside of the package